README.md

```diff
@@ -208,7 +208,7 @@ or curious about UI/Design in general,
 you _might_ find certain features familiar.
 You may think:
 "_note-taking similar to Evernote_",
-"_collaborative editing like Google Docs_"
+"_collaborative editing like Google Docs_"x§
```

Comment:
```suggestion
"_collaborative editing like Google Docs_"
```

Reply:
Assuming this was an unintentional add

Reply:
yeah, must have been [Nala](https://github.com/nelsonic/nelsonic.github.io/issues/379) walking on my keyboard when I was AFK ... 🐈 😉